### ISSUE TITLE
screen: error 127 on launch, fresh babun install  

### ISSUE THREAD

**Issue by quentin-ol** [babun/babun#807]
Hello,

On a fresh babun install, I installed screen: 
```bash
pact install screen
```
when I run it, I get the error 127.

It might not be relevant, but this was the first run of babun, i.e. first time run after the installation and the startup-routine checks reported this:
```shell
/usr/bin/id: cannot find name for group ID 10513
/usr/bin/id: cannot find name for group ID 10513
Executing daily babun check:
Source consistent [OK]
Prompt speed      [OK]
File permissions  [OK]
Connection check  [OK]
Update check      [OK]
Cygwin check      [OK]
Welcome to babun - the Windows shell you will love!
You have got a lot of things under the hood here!
```

**Comment by Aeres-u99** [comment]
ugh... I remember installing screen was a truly painful experience , I have installed tmux ,did that work out ? And also I remember using Cygwin's setup to install screen.
Besides all this , try running update.bat and rebase.bat .

**Comment by quentin-ol** [comment]
update: this issue is not related to the screen package nor babun.
`/usr/bin/id: cannot find name for group ID 10513` should not be ignored as it seemed to be the cause of my error.
I followed the instruction located [here](https://serverfault.com/questions/666639/cygwin-startup-error-usr-bin-id-cannot-find-name-for-group-id).
In case this page goes down:
I issued the following command in babun (read Cygwin)
```bash
mkgroup -l > /etc/group
mkgroup -c >> /etc/group
```
then I did:
```bash
babun install
pact remove screen
pact install screen
```
screen is now working.

bottom line of the story:
I'm on a corporate environment with an Active Directory and those steps seem to be mandatory (don't ask me why). It might be worth to document that in the readme.md?

**Comment by thomasklash** [comment]
Make a pull request

**Comment by klaussmit88** [comment]
I am trying to follow the steps suggested, but I get the following. Please help. 
/usr/bin/id: cannot find name for group ID 10513
/usr/bin/id: cannot find name for group ID 10513
{ ~ }  »  mkgroup -l > /etc/group                                                               ~
{ ~ }  » mkgroup -c >> /etc/group                                                               ~
mkgroup (165): [1789] The trust relationship between this workstation and the primary domain failed.
{ ~ }  »

**Comment by Dumbaz** [comment]
The ~127 Errors usually show up when update.bat in the babun folder had never been run after intial installation.
